package.json

```diff
@@ -14,9 +14,11 @@
   },
   "dependencies": {
     "html-webpack-plugin": "^2.28.0",
+    "leaflet": "^1.0.3",
```

Comment:
Let me quote this:
```
dependencies are required to run, devDependencies only to develop, e.g.: unit tests, Coffeescript to Javascript transpilation, minification, ...
```
https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies